test/Microsoft.AspNet.Mvc.ModelBinding.Test/Binders/TypeConverterModelBinderTest.cs

```diff
@@ -71,8 +71,7 @@ public class TypeConverterModelBinderTest
         public async Task BindModel_Error_FormatExceptionsTurnedIntoStringsInModelState()
         {
             // Arrange
-            var message = "The parameter conversion from type 'System.String' to type 'System.Int32' failed." +
-                " See the inner exception for more information.";
+            var message = "The value 'not an integer' is not valid for theModelName.";
             var bindingContext = GetBindingContext(typeof(int));
             bindingContext.ValueProvider = new SimpleHttpValueProvider
```

Comment:
Is this the only unit test that covers this behavior? Does anything else call `AddModelErrorBasedOnExceptionType`?